keps/sig-release/2572-release-cadence/README.md

```diff
@@ -0,0 +1,320 @@
+also possible to include more shadows if the role leads conclude that this is
+appropriate.
+
+### Risks and Mitigations
+
+#### Concentrating risk
+
+In theory a reduced release cadence will cause more changes for every release.
+This means that there will be an increased risk, which would usually be split
+up into 4 dedicated milestones rather than 3.
+
+SIG Release cannot mitigate this risk directly, but is able to track and
+influence it during each release cycle. It's the responsibility of SIG Release,
+together with SIG Testing and SIG Architecture, to identify new gaps and issues
+in the release cadence and mitigate them on a case-by-case basis.
+
+#### Attention to tests
+
+This KEP does not propose any change to the release cycle itself and assumes
+that the same periods for Code and Test Freeze. Assuming that, there is an
+increased risk for flakes and test failures. It will be the responsibility of
+SIG Release to mitigate this, together with the CI signal role. If we speak
+about an overall release cycle enhancement of 3-4 weeks, then we believe that
+SIG Release is able to mitigate this risk over multiple releases.
+
+#### Attention to dependencies
+
+Having fewer releases will introduce the risk of missing dependencies — for
+example, Golang upgrades. This has to be mitigated on a case-by-case basis, in
+the same way as it is being done right now.
+
+#### Feature graduation
+
+Research discovered that only 5% of Kubernetes features advanced from Alpha to
+GA in the minimum 3 releases. However, the same research showed that reminders
+from the Release Team played a critical role in advancement of more than 50% of
+features. With an increased release cycle, this reminder activity can be
+expected to slow down. As such, advancement will need to be mitigated by making
+sure that SIGs keep track of their feature enhancement in more detail.
+
+## Design Details
+
+### Schedule Policy
+
+The exact details of the release schedule are created by the team leads before
+the actual cycle begins. SIG Release only provides a lightweight policy, which
+is defined as:
+
+1. The first Kubernetes release of a year should start at the second or third
+   week of January to provide people more room after coming back from the
+   Christmas holidays.
+
+2. The last Kubernetes release of a year should be finished by the middle of
+   December.
+
+3. A Kubernetes release cycle has a length of of ~15 weeks.
+
+4. Events like KubeCon will be considered as blocked from development or
+   decision-making from the SIG release perspective. SIG Release will also
+   consider the week before and after the event in the same way.
+
+5. An explicit SIG release break of at least two weeks between each cycle will
+   be enforced.
+
+   This does not mean that zero development can happen during that time.
+   Rather, SIG Release will use this time to do the release retrospective and
+   plan for the next cycle.
+
```

Comment:
Please add an example here. I'm looking for something like the following (my interpretation of what's written above:

## k8s releases, 2022

| Year Week Number | Release Number | Release Week |
| - | - | - |
| 2 | 1 | 1 (Jan. 10) |
| 16 | 1 | 15 (Apr. 18) |
| 18 | 2 | 1 (May 2) |
| 33 | 2 | 15 (Aug. 15) |
| 35 | 3 | 1 (Aug. 29) |
| 50 | 3 | 15 (Dec. 12) |

The most interesting thing I found when putting this table together is that there was no "easy" way to block off KubeCons.

Reply:
Since the intent is implement this starting with 1.22, would a chart showing this year and 2022 be useful?

Reply:
Er, whose intent?  I'd assumed (and I think everyone else did too) that, given that this KEP isn't even merged yet, that we plan to implement 4-month releases starting with 1.25 in 1Q 2022.  Starting with 4-month releases in 2021 would be very confusing for everyone involved, unless it somehow just happens to line up with the calendar we want if we started in 1.23.

Reply:
Leaving aside any time that our project needs to prepare for a change to release cadence, consider that we have downstreams.

Reply:
can we resolve this conversation now?

Reply:
We *still* need a calendar in the KEP, even if (I think) we have consensus on that calendar now.

Reply:
I’ve added a table with dates for 2021 and 2022, but just start and end. What granularity are you looking for, and does it need to move to a different location/have a specific header for table of contents?

Reply:
i am happy with what's in the KEP so far. wrapping this up.